src/style.css

```diff
@@ -0,0 +1,116 @@
+@import "../../layout_stars/src/style.css";
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.card {
+  display: flex;
+  padding: 0 16px;
+  width: 200px;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin: 32px 3px;
```

Comment:
Usually the distance between the rows of elements is set either by the lower or upper margin for each element in the entire project.